### ISSUE TITLE
Delete link to github when using '@'

### ISSUE THREAD

**Issue by midoo1992** [mkdocs/mkdocs#2279]
When i use '@', it become automatically a link to github.
Exemple:
When i write @api, this word have a link to "http://github.com/api"
Can someone help me to delete the link ?

**Comment by facelessuser** [comment]
What extension are you using to translate `@` to github links? Are you using [MagicLink](https://facelessuser.github.io/pymdown-extensions/extensions/magiclink/) or something else?

If you are using MagicLink, you can exclude certain names from the detection https://facelessuser.github.io/pymdown-extensions/extensions/magiclink/#user-excludes.

You can also escape the `@` to avoid parsing it has a link: `\@api`.

In short, this is not a MkDocs question but a Markdown parsing question, specifically a question tied to a particular Markdown extension (whichever it is that you use).

**Comment by facelessuser** [comment]
It should be noted, that if you are using MagicLink, this could be excluded by default. You would need to create an issue over at that repository.

**Comment by oprypin** [comment]
This doesn't happen by default, and the original reporter did not provide additional information.
